public/stylesheets/application.css

```diff
@@ -0,0 +1,10 @@
+$govuk-assets-path: "/report-quarterly/income-and-expenses/sign-up/eligibility/assets/lib/govuk-frontend/govuk/assets/";
+$hmrc-assets-path: "/report-quarterly/income-and-expenses/sign-up/eligibility/assets/lib/hmrc-frontend/hmrc/";
+
+@import "lib/govuk-frontend/govuk/all";
+@import "lib/hmrc-frontend/hmrc/all";
+
+.app-reference-number {
+  display: block;
+  font-weight: bold;
+}
```

Comment:
This isn't in the correct place and the file type is incorrect, please refer to the eligibility frontend as a reference as to where this file should go and be called